fearless/Modules/Staking/SelectValidatorsFlow/SelectedValidatorList/SelectedValidatorListViewController.swift

```diff
@@ -0,0 +1,240 @@
+    }
+
+    override func loadView() {
+        view = SelectedValidatorListViewLayout()
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        setupTable()
+        setupNavigationBar()
+        setupProceedButton()
+
+        applyLocalization()
+
+        presenter.setup()
+    }
+
+    // MARK: - Private functions
+
+    private func setupTable() {
+        rootView.tableView.dataSource = self
+        rootView.tableView.delegate = self
+        rootView.tableView.registerClassForCell(SelectedValidatorCell.self)
+        rootView.tableView.registerHeaderFooterView(withClass: SelectedValidatorListHeaderView.self)
+    }
+
+    private func setupNavigationBar() {
+        let rightBarButtonItem = UIBarButtonItem(
+            title: "",
+            style: .plain,
+            target: self,
+            action: #selector(tapEditButton)
+        )
+
+        let attributes: [NSAttributedString.Key: Any] = [
+            .foregroundColor: R.color.colorWhite()!,
+            .font: UIFont.h5Title
+        ]
+
+        rightBarButtonItem.setTitleTextAttributes(attributes, for: .normal)
+        rightBarButtonItem.setTitleTextAttributes(attributes, for: .highlighted)
+
+        navigationItem.rightBarButtonItem = rightBarButtonItem
+    }
+
+    private func setupProceedButton() {
+        rootView.proceedButton.addTarget(self, action: #selector(tapProceedButton), for: .touchUpInside)
+        updateProceedButton()
+    }
+
+    private func updateEditButton() {
+        if rootView.tableView.isEditing {
+            navigationItem.rightBarButtonItem?.title = R.string.localizable
+                .commonDone(preferredLanguages: selectedLocale.rLanguages)
+        } else {
+            navigationItem.rightBarButtonItem?.title = R.string.localizable
+                .commonEdit(preferredLanguages: selectedLocale.rLanguages)
+        }
+    }
+
+    private func updateProceedButton() {
+        let cellViewModels = viewModel?.cellViewModels ?? []
+
+        let buttonTitle: String
+        let enabled: Bool
+        let fillColor: UIColor
+
+        if cellViewModels.count > selectedValidatorsLimit {
+            enabled = false
+            fillColor = R.color.colorDarkGray()!
+            buttonTitle = R.string.localizable
+                .stakingCustomProceedButtonDisabledTitle(
+                    selectedValidatorsLimit,
+                    preferredLanguages: selectedLocale.rLanguages
+                )
+
+        } else {
+            enabled = true
+            fillColor = R.color.colorAccent()!
+            buttonTitle = R.string.localizable
+                .commonContinue(
+                    preferredLanguages: selectedLocale.rLanguages
+                )
+        }
+
+        rootView.proceedButton.triangularedView?.fillColor = fillColor
+        rootView.proceedButton.imageWithTitleView?.title = buttonTitle
+        rootView.proceedButton.isEnabled = enabled
```

Comment:
I think it is also changes alpha for disabled state and it not 1 by default. So it is still better to refactor ```applyDisabledStyle```